components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -342,7 +343,7 @@ class RhsRootPost extends React.PureComponent {
         if (!isPostDeleted && this.props.post.type !== Constants.PostTypes.FAKE_PARENT_DELETED) {
             dotMenuContainer = (
                 <div
-                    ref='dotMenu'
+                    ref={this.getDotMenuRef}
```

Comment:
not really a problem, but you can link the ref directly. the getter is mostly for anything outside the component
```suggestion
                    ref={this.dotMenuRef}
```

Reply:
This is also changed.